README.md

```diff
@@ -1,2 +1,34 @@
 # scmd-narcolepsy
-Investigation of medicines used in secondary care to treat excess daytime sleepiness caused by narcolepsy
+Investigation of medicines used in Primary (English Prescribing Data) care & secondary (Secondary Care Medicines Data) care to treat narcolepsy
+
+Description of R scripts:
+  - "OctGitHubFinal.R" - Data cleaning + analysis of secondary care data (SCMD). Largely adapated from initial code provided by DataLab team
```

Comment:
use relative file paths here and backticks instead of quotation marks to specify the files. also no need to say that the code came from us originally I think. I'd keep it simple and just say something like this:

```suggestion
  - `scripts/OctGitHubFinal.R` - Data cleaning and analysis of secondary care data (SCMD)
```